themes/custom/droopler_theme/scss/layout/_header.scss

```diff
@@ -16,8 +16,23 @@ header.header {
     max-height: $header-height - (2 * $navbar-padding-y);
   }
   .region-primary-menu {
-    .navbar {
-      padding: $navbar-padding-y 0;
+    .region-we-mega-menu {
+      margin-bottom: 0;
+      .navbar {
+        padding: $navbar-padding-y 0;
+        &.navbar-we-mega-menu {
+          @include media-breakpoint-down(md) {
+            width: 250px;
```

Comment:
it's temporary solution to make it look tolerably, we do not have designs for menu for now.